src/main/resources/edu/cornell/kfs/ksr/document/datadictionary/SecurityRequestDocument.xml

```diff
@@ -60,10 +60,6 @@
   <bean id="SecurityRequestDocument-requestPerson.principalName" parent="SecurityRequestDocument-requestPerson.principalName-parentBean"/>
   <bean id="SecurityRequestDocument-requestPerson.principalName-parentBean" abstract="true" parent="PersonImpl-principalName">
     <property name="name" value="requestPerson.principalName"/>
-    <property name="validationPattern"><null /></property>
```

Comment:
Was this causing an issue as well? Based on the jira description only an issue with the department code was reported.
I see a similar change was made for the principalNameForSearch. Was that also needed? Thank you.

Reply:
While I was preparing the changes for this, I noticed that several KSR attributes were using the "validCharactersConstraint" setup instead of the "validationPattern" setup. I went ahead and switched the other attributes from the constraint setup to the pattern setup for consistency, especially since that transition seemed to resolve the validation problems on the department code. This also helped bring our configuration more in line with base code, which is almost entirely using the "validationPattern" setup throughout the various modules.

Reply:
I think that is already set up and inherited from KimBaseBeans-principalName so that is why this is not needed? Thank you.

Reply:
Yes, for the other cases in this file, they have KimBaseBeans-principalName as an ancestor and are inheriting its validation pattern. Sorry for the confusion.

Reply:
Thank you.